deployment_cdk/.specific/deployment_options.adoc

```diff
@@ -1,8 +1,8 @@
-// Edit this placeholder text to accurately describe your architecture.
+// Edit this placeholder text as necessary to describe the deployment options.
 
-This Quick Start provides two deployment options:
+This Quick Start provides the following deployment options:
 
 * http://qs_launch_permalink[Deploy {partner-product-short-name} into a new VPC]. This option builds a new AWS environment consisting of the VPC, subnets, NAT gateways, security groups, bastion hosts, and other infrastructure components. It then deploys {partner-product-short-name} into this new VPC.
 * http://qs_launch_permalink[Deploy {partner-product-short-name} into an existing VPC]. This option provisions {partner-product-short-name} in your existing AWS infrastructure.
 
-The Quick Start provides separate templates for these options. It also lets you configure Classless Inter-Domain Routing (CIDR) blocks, instance types, and {partner-product-short-name} settings.
+This Quick Start provides separate templates for these options. It also lets you configure Classless Inter-Domain Routing (CIDR) blocks, instance types, and {partner-product-short-name} settings.
```

Comment:
Now that the bullets above this paragraph include the launch permalinks, I suggest that we move this information (about the templates) into the paragraph preceding the bullets. That way, people learn what there is to learn about the options before they click one.